src/app/test_executive/zkapps.ml

```diff
@@ -242,7 +243,8 @@ module Make (Inputs : Intf.Test.Inputs_intf) = struct
                       , zkapp_command_invalid_nonce
                       , zkapp_command_insufficient_funds
                       , zkapp_command_insufficient_replace_fee
-                      , zkapp_command_insufficient_fee ) =
+                      , zkapp_command_insufficient_fee
+                      , zkapp_command_single_account_update ) =
```

Comment:
nit: could you rename this to something more meaningful- `zkapp_command_cross_network_replay` or something?